roles/install_dbserver/tasks/linux_coredump.yml

```diff
@@ -29,11 +29,21 @@
   become: true
   no_log: "{{ disable_logging }}"
 
-- name: Install debug packages
+- name: Install debuginfo helper packages
   ansible.builtin.package:
-    name: edb-as{{ pg_version }}-*-debuginfo
+    name: yum-utils
     state: present
   become: true
-  no_log: "{{ disable_logging }}"
+
+- name: Install debug packages
+  ansible.builtin.shell:
```

Comment:
does this task work with the command module instead of shell? The linting doesn't like the shell

Reply:
yeah it should, will make that change.

Reply:
forced push, let's see how the tests do...